src/const/urls.ts

```diff
@@ -3,7 +3,7 @@ import type { SitemapPage } from '@/types/sitemap';
 export const JUMPER_URL = 'https://jumper.exchange';
 export const LIFI_URL = 'https://li.fi';
 export const EXPLORER_URL = 'https://explorer.li.fi';
-export const DISCORD_URL = 'https://discord.gg/lifi';
+export const DISCORD_URL = 'https://discord.gg/jumperexchange';
```

Comment:
Do we want to use this export here everywhere instead of redefining the URL everytime?